### ISSUE TITLE
Does MultiROM support system-as-root?

### ISSUE THREAD

**Issue by ARTI5T** [vasishath/multirom#22]

**Comment by vasishath** [comment]
Yes multirom supports the android 10 version of system-as-root. In addition to that, it can also boot a system-as-root secondary rom on a non system-as-root device.

**Comment by Helloindiaa** [comment]
Yes
But not at same time in primary and secondary roms
I tried Android 10 and 11 or 11 and 10. Of i flash magisk in secondary device formats automatically so
Better to use root only in primary rom
